### ISSUE TITLE
Article text removed from arstechnica[dot]com [Issue #3284 closed for lack of information]

### ISSUE THREAD

**Issue by gatoraitor** [easylist/easylist#3524]
New issue after #3284 was closed for not including enough information.  

Sorry, I guess I thought with the rule it would be clear that it's going to remove an entire tag if it's a match.  I've included a couple that I found.  I can grab some more if needed, I usually just notice it when I'm reading or someone points it out in the comments.

The amount of text used in the link is what makes the difference between when it isn't really a problem and when something won't even make sense, but it's more of an annoyance than it is helpful in this case.

### List the website(s) you're having issues:

`arstechnica[dot]com`

### What happens?

Simple article text is removed when it's a link to amazon.com.  Depending on the size of the text in the link this can either be minimal or make a section of text difficult to read and understand.

### List Subscriptions you're using:

Easylist English

### Your settings

Only using Adblock Plus

- OS/version: macOS Mojave 10.14.4
- Browser/version: Chrome Version 73.0.3683.103 (Official Build) (64-bit)
- Adblock Extension/version: Adblock Plus 3.5.1

### Other details:

Looks like it's coming from this filter specifically:

```
9to5google.com,9to5mac.com,9to5toys.com,ancient-origins.net,aol.co.uk,arstechnica.com,catholicculture.org,electrek.co,gamenguide.com,ign.com,indianexpress.com,lewrockwell.com,livescience.com,macworld.co.uk,marineelectronics.com,marinelink.com,marinetechnologynews.com,maritimejobs.com,maritimeprofessional.com,maritimepropulsion.com,myanimelist.net,oneangrygamer.net,parentherald.com,segmentnext.com,ssbcrack.com,yachtingjournal.com##a[href^="https://www.amazon."][href*="tag="]
```
### Examples: 

__URL:__
https://arstechnica.com/science/2019/05/no-someone-hasnt-cracked-the-code-of-the-mysterious-voynich-manuscript/

__Result:__
_Full sentence:_ 
"Given that Gibbs' main claim to fame before this article was a series of books about [how to write and sell television screenplays](https://www.amazon.com/dp/1444167596/?tag=arstech20-20), it seems that his goal in this research was probably to sell a television screenplay of his own."

_Truncated Sentence:_  
"Given that Gibbs' main claim to fame before this article was a series of books about , it seems that his goal in this research was probably to sell a television screenplay of his own."

__URL:__
https://arstechnica.com/science/2019/04/who-had-the-most-merciful-death-on-game-of-thrones-science-has-an-answer/

__Result:__
_Full sentence:_ 
These and other scintillating topics are discussed in a forthcoming book by physicist (and uber-fan) Rebecca Thompson, [Fire, Ice, and Physics: The Science of Game of Thrones](https://www.amazon.com/dp/0262043076/?tag=arstech20-20). 

_Truncated Sentence:_  
These and other scintillating topics are discussed in a forthcoming book by physicist (and uber-fan) Rebecca Thompson, .

**Comment by ryanbr** [comment]
aren't the amazon links just text ads?

**Comment by gatoraitor** [comment]
I don't think of them as ads like I do with ads that actually have a resource impact on my browsing.  It's just a link I'm never going to click on.  I'd be all for the removal of the link, but removing text from an article is confusing.  At the end of the day I still have sources that I trust to discuss and recommend various products, games, movies, etc. and I'd rather be able to read about it then worry about it linking to a place to buy it.

If every simple link to an e-commerce site is treated as an ad, then browsing is going to get pretty hairy I worry.

**Comment by ryanbr** [comment]
Amazon is so commonly used, its considered an advert (outside of amazons own site)

**Comment by llacb47** [comment]
You shouldn't delete text from an article just because it links to Amazon.

**Comment by Wxfisch** [comment]
This is an ongoing issue, the affiliate links at Ars Technica are not in-line ads, they are a part of the article text that in some cases link out to Amazon (or Walmart, or wherever) to buy the product being discussed. This would be no different than if they simply put a non-affiliate link to the product. Add in that Ars actually discloses their use of affiliate links and this seems like an overstep on the part of EasyList.

**Comment by Clearnine** [comment]
Sorry, but this is just incorrect:

> Amazon is so commonly used, its considered an advert (outside of amazons own site)

Considered by who? A hyperlink, or a list of hyperlinks are **links**.  **Not ads**, and **not tracking**, especially when the links are completely in context and in no way disguised as anything other than what they are; links to product pages.  Amazon doesn't know what pages you've visited based on the presence of these links, and cannot track you.  This appears to be a wholly arbitrary decision.

**Comment by imgx64** [comment]
@ryanbr I don't understand why this issue is closed. *Text from the article itself* is being deleted. It's not just a standalone link.

URL: https://arstechnica.com/gadgets/2019/07/can-disneys-circle-really-deliver-a-porn-free-internet/2/

With EasyList:
![We tested the , but a is available now.](https://user-images.githubusercontent.com/318668/61606977-6ea4c300-ac55-11e9-9816-89569e84b57a.png)

Without EasyList:
![We tested the first-generation Circle, but a newer version is available now.](https://user-images.githubusercontent.com/318668/61607062-ae6baa80-ac55-11e9-9a39-4595c129180b.png)
